ruff_cli/src/commands/linter.rs

```diff
@@ -0,0 +1,57 @@
+use crate::args::HelpFormat;
+use itertools::Itertools;
+use ruff::registry::{Linter, LinterCategory, RuleNamespace};
+use serde::Serialize;
+use strum::IntoEnumIterator;
+
+pub fn linter(format: HelpFormat) {
+    match format {
+        HelpFormat::Text => {
+            for l in Linter::iter() {
```

Comment:
I know this is really annoying of me but can we use `linter` in lieu of `l` here, and in general use full names rather than abbreviations?

Reply:
I went ahead and changed this, hopefully that's ok!

Reply:
Yes, that's alright :) Thanks!